### DISCUSSION TITLE
2024/04/19/opl_ui/

### DISCUSSION THREAD

**Discussion by giscus** [Guailoudou/pages#6] [Announcements]
# 2024/04/19/opl_ui/

乖漏斗博客

https://blog.gldhn.top/2024/04/19/opl_ui/

**Comment by wjsxgd** [comment]
可以开机自动启动软件，并自动启动吗？

**Reply by Guailoudou** [reply]
后续可能会以增加启动参数的形式实现可自行部署开机自启

**Reply by Guailoudou** [reply]
对了现在支持了哦，可以直接在设置启用

**Comment by xiaofengjun0879** [comment]
问问未来会上架微软商店吗

**Reply by Guailoudou** [reply]
感觉不会

**Comment by 45355993** [comment]
up我想用这个软件联机流放者柯南，一直连不上

**Reply by Guailoudou** [reply]
这个你可以试试那个新的组网，或许能行

**Comment by a12387** [comment]
请问能不能支持关闭窗口后保持程序在后台运行

**Reply by Guailoudou** [reply]
阿巴，刚才看邮箱才看到这消息，我觉得这功能可以有

**Comment by XIAO-HA-114514** [comment]
能更新一下安卓版的OPL吗
安卓端OPL没法作为房主开房间

**Reply by Guailoudou** [reply]
可以作房主的，你不输入uid和端口直接启动就行
